scripts/shared/inlinedHostConfigs.js

```diff
@@ -184,6 +184,33 @@ module.exports = [
     isFlowTyped: true,
     isServerSupported: true,
   },
+  {
+    shortName: 'dom-edge-esm',
+    entryPoints: ['react-server-dom-esm/server.edge'],
+    paths: [
```

Comment:
I didn't quite understand what these paths actually meant, so just based them on what I found `react-server-dom-esm/server.node` and `react-server-dom-webpack/server.edge` did. Happy to make any changes here if I messed that up!